django/contrib/admin/sites.py

```diff
@@ -63,6 +64,21 @@ def __init__(self, name='admin'):
         self.name = name
         self._actions = {'delete_selected': actions.delete_selected}
         self._global_actions = self._actions.copy()
+        all_sites.add(self)
+
+    def check(self):
+        """
+        Run the system checks of the underlying admin objects.
```

Comment:
how about: "Run the system checks on all ModelAdmins, except if they aren't customized at all."